transcendence-app/test/auth/auth.e2e-spec.ts

```diff
@@ -1,11 +1,11 @@
 import { Test, TestingModule } from '@nestjs/testing';
-import { AuthController } from './auth.controller';
+import { AuthController } from '../../src/auth/auth.controller';
 import { HttpStatus, INestApplication } from '@nestjs/common';
-import { AuthModule } from './auth.module';
+import { AuthModule } from '../../src/auth/auth.module';
 import * as request from 'supertest';
 import { ConfigModule, ConfigService } from '@nestjs/config';
-import { validate } from '../config/env.validation';
-import { OAuth42Config } from './oauth42-config.interface';
+import { validate } from '../../src/config/env.validation';
+import { OAuth42Config } from '../../src/auth/oauth42-config.interface';
```

Comment:
I think we will need soon (or already?) some way to shorten the import paths. Something like [this](https://getfrontend.tips/shorten-import-paths-in-typescript/)

Reply:
This is interesting, thank you. We could make a new task for this because it requires changing almost all the files if we want to use this syntax.